src/routes/dashboard.tsx

```diff
@@ -0,0 +1,30 @@
+import { MetricBoxList } from "../components";
+
+export const Dashboard: React.FC = () => {
+  return (
+    <MetricBoxList
```

Comment:
You may want to center this component just in case the screen is wider than 390px so it doesn't hug the left side of the screen. I don't see many phones being wider than that so probably not a big deal.